testcafe/tests/pages/drive-model-public.js

```diff
@@ -0,0 +1,118 @@
+import { Selector, t } from 'testcafe'
+import {
+  getElementWithTestId,
+  isExistingAndVisibile,
+  getPageUrl,
+  goBack,
+  getResponseStatusCode
+} from '../helpers/utils'
+import osHomedir from 'os-homedir'
+import fs from 'fs'
+
+export default class PublicDrivePage {
+  constructor() {
+    //loading
+    this.content_placeholder = Selector(
+      '[class*="fil-content-file-placeholder"]'
+    )
+
+    //Logo
+    this.logo = Selector('.coz-nav-apps-btns-home')
+
+    //Toolbar - Action Menu
+    this.toolbar_files_public = getElementWithTestId('fil-toolbar-files-public')
```

Comment:
Wen we use camel-case for this property too?

Reply:
can ? :d

Reply:
I don't even know how you can type "Wen" instead of "Can" 🤦‍♂️. But yes.